tooling/v8-snapshot/src/generator/snapshot-generator.ts

```diff
@@ -462,7 +462,7 @@ export class SnapshotGenerator {
 
     // 2. Run the `mksnapshot` binary providing it the path to our snapshot
     //    script
-    const args = [this.snapshotScriptPath, '--output_dir', this.snapshotBinDir]
+    const args = [this.snapshotScriptPath, '--output_dir', this.snapshotBinDir, '--no-use-ic']
```

Comment:
See https://issues.chromium.org/issues/345280736#comment12 for context - we should add this inline as a comment.